motoman_hc20_support/config/opw_parameters_hc20dtp.yaml

```diff
@@ -0,0 +1,20 @@
+#
+# Parameters for use with IK solvers which support OPW (Ortho-Parallel Wrist)
+# kinematic configurations, as described in the paper "An Analytical Solution
+# of the Inverse Kinematics Problem of Industrial Serial Manipulators with an
+# Ortho-parallel Basis and a Spherical Wrist" by Mathias Brandstötter, Arthur
+# Angerer, and Michael Hofbaur (Proceedings of the Austrian Robotics Workshop
+# 2014, 22-23 May, 2014, Linz, Austria).
+#
+# The moveit_opw_kinematics_plugin package provides such a solver.
+#
+opw_kinematics_geometric_parameters:
+    a1:  0.000
+    a2:  0.000
+    b:   0.000
```

Comment:
@EricMarcil @cjue: before e7a2ee7d this read `-0.0`. I'm assuming that was copy-pasta and removed the `-` sign.